src/regression_tracker.py

```diff
@@ -83,6 +83,7 @@ def _create_regression(self, failed_node, last_pass_node):
         regression['state'] = 'done'
         error = self._collect_errors(failed_node)
         regression['data'] = {
+            'fail_node_kind': failed_node['kind'],
```

Comment:
Is this to distinguish if the regression is for a job node or a test node?

Reply:
Okay. Shouldn't this field be added [here](https://github.com/kernelci/kernelci-core/blob/main/kernelci/api/models.py#L506) first?